.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+    "env": {
+        "commonjs": true,
+	"node": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
+    "globals": {
```

Comment:
Do we really use either of these global "variables"? If not, you can remove the `globals` section.